libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/placeslistonmap/PlacesListOnMapScreen.kt

```diff
@@ -44,6 +47,7 @@ class PlacesListOnMapScreen @UiThread constructor(
     private val placesListOnMapManager = PlacesListOnMapManager(placesProvider)
 
     init {
+        attachCreated(searchCarContext.carRouteRequest, searchCarContext.carPlaceSearch)
```

Comment:
the carRouteRequest is already registered here in the DefaultLifecycleObserver

Will you move this, trying to avoid `@OptIn(ExperimentalPreviewMapboxNavigationAPI::class)` wherever we can

Reply:
Oh, looks like it is not necessary then, I removed the commit. The issue was present in `0.12.0` release, but it is already fixed on the latest commit in `main`.